Source/WebCore/editing/EditorCommand.cpp

```diff
@@ -1771,8 +1755,6 @@ static const CommandMap& createCommandMap()
         { "Print"_s, { executePrint, supported, enabled, stateNone, valueNull, notTextInsertion, doNotAllowExecutionWhenDisabled } },
         { "Redo"_s, { executeRedo, supported, enabledRedo, stateNone, valueNull, notTextInsertion, doNotAllowExecutionWhenDisabled } },
         { "RemoveFormat"_s, { executeRemoveFormat, supported, enabledRangeInEditableText, stateNone, valueNull, notTextInsertion, doNotAllowExecutionWhenDisabled } },
-        { "ScrollPageBackward"_s, { executeScrollPageBackward, supportedFromMenuOrKeyBinding, enabled, stateNone, valueNull, notTextInsertion, doNotAllowExecutionWhenDisabled } },
```

Comment:
What are the implications of removing these? They've been here for a while.

Reply:
As far as I can tell, these are here because if you are in a content-editable div with lots of text, and you do Option+Shift+Page Down (or something like that), it will both move the caret down, and it will also do a page down scroll. 

So, because of that, the current behavior is that it goes into `defaultKeyboardEventHandler`, and first checks if there is an editing command, and in this case there is, so it does that command (which involves moving the caret and scrolling), and returns.

The new behavior changes the editing command so that the editing part only modifies the caret, and `defaultKeyboardEventHandler` does the scrolling part instead.

This behavior is covered by a layout test (LayoutTests/editing/input/option-page-up-down.html), and that passes, and to me the logic makes sense too, so I don't expect doing this will cause any problems.